### PR TITLE
Add first PHP creedengo plugin 2.0.0 from PHP ecocode plugin 

### PR DESCRIPTION
**Pull request by dedece35**
Hi,

like already done in PR https://github.com/SonarSource/sonar-update-center-properties/pull/640 for python plugin, 
we renamed our ecocode-php plugin to creedengo-php plugin.
This is the first release of this creedengo php plugin.

You can check the changelog here :
https://github.com/green-code-initiative/creedengo-php/releases/tag/2.0.0

The corresponding SonarCloud project : https://sonarcloud.io/project/overview?id=green-code-initiative_creedengo-php

Please feel free to ask questions about this first integration.

Thank you

Regards

### CODE REVIEW

File: ecocodephp.properties [hunk 1]
```diff
@@ -1,8 +1,8 @@
 category=External Analysers
 description=Provide a list of static code analyzers (for PHP language) to highlight code structures that may have a negative ecological impact: energy and resources over-consumption, "fatware", shortening terminals' lifespan, etc.
```
Comment (ganncamp) [inline ecocodephp.properties:2]: Please add text about the deprecation here.

File: creedengophp.properties [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+description=Provide a list of static code analyzers (for PHP language) to highlight code structures that may have a negative ecological impact: energy and resources over-consumption, "fatware", shortening terminals' lifespan, etc.
+homepageUrl=https://github.com/green-code-initiative/creedengo-php
+publicVersions=2.0.0
+archivedVersions=
+
+defaults.mavenGroupId=org.green-code-initiative
+defaults.mavenArtifactId=creedengo-php-plugin
+
+2.0.0.description=Initial Marketplace publication of plugin after renaming from "ecocode-php"
+2.0.0.sqVersions=[9.9,10.6.0]
```
Comment (ganncamp) [inline creedengophp.properties:11]: This is a brand new release. Are you sure you don't want it to be usable in the latest versions?

**Review by ganncamp — CHANGES_REQUESTED** [review]

File: creedengophp.properties (hunk 2 above)
Comment (dedece35) [reply creedengophp.properties:11]: I will test our plugin with the last plugin I will update if OK.
thank you for feedback

File: ecocodephp.properties (hunk 1 above)
Comment (dedece35) [reply ecocodephp.properties]: update done

File: creedengophp.properties (hunk 2 above)
Comment (dedece35) [reply creedengophp.properties:11]: finally, it's ok for 10.7 version but not ok with 10.8 for now.
If you doesn't mind, I prefer to manage 10.7 and 10.8 compatibilities inside next version.
thank you.

Comment (ganncamp) [reply creedengophp.properties:11]: It's up to you.

**Pull request merged**